index.html

```diff
@@ -1051,16 +1051,22 @@ <h3>Data delivery</h3>
               and {{PressureRecord/[[Time]]}} set to |timestamp|.
             </li>
             <li>
-              If |observer|.{{PressureObserver/[[AfterPenaltyRecordMap]]}}[source] exists,
-              set observer.{{PressureObserver/[[AfterPenaltyRecordMap]]}}[source] to |record|
-              and [=iteration/continue=].
+              Increase |observer|.{{PressureObserver/[[ChangesCountMap]]}}[|source|].
             </li>
             <li>
               If running [=passes rate obfuscation test=] with |observer| and |source| returns false:
               <ol>
+                <li>
+                  If |observer|.{{PressureObserver/[[AfterPenaltyRecordMap]]}}[source] exists,
+                  set observer.{{PressureObserver/[[AfterPenaltyRecordMap]]}}[source] to |record|
+                  and [=iteration/continue=].
+                </li>
                 <li>
                   Set |observer|.{{PressureObserver/[[AfterPenaltyRecordMap]]}}[|source|] to |record|.
                 </li>
+                <li>
+                  Reset |observer|.{{PressureObserver/[[ChangesCountMap]]}}[|source|].
```

Comment:
"Reset an integer" sounds weird and is not really defined. "Set to 0" sounds clearer.

Reply:
agree